Makefile

```diff
@@ -22,14 +22,22 @@ endif
 
 .PHONY: all docs test itest
 
-docs:
-	./docs.sh
+docs: .paasta/bin/activate
```

Comment:
Can you ship these makefile changes in a different PR so they can be bike-sheded independently?